examples/pylab_examples/print_stdout.py

```diff
@@ -1,13 +1,9 @@
-#!/usr/bin/env python
-# -*- noplot -*-
-# print png to standard out
```

Comment:
The second line there is a Sphinx directive (Note: I might have my
terminology wrong) that tells the documentation builder to not bother
running this script to generate an output image, because it won't work. We
also use this directive for the interactive examples like the pong clone.

On Sat, Aug 1, 2015 at 11:08 PM, Eric Ma <notifications@github.com> wrote:

> In examples/pylab_examples/print_stdout.py
> <https://github.com/matplotlib/matplotlib/pull/4845#discussion_r36039118>:
>
> > @@ -1,13 +1,9 @@
> > -#!/usr/bin/env python
> > -# -*- noplot -*-
> > -# print png to standard out
>
> What does the 2nd line tell us? Btw, I'm adding them back anyways, but
> just curious.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4845/files#r36039118>.
>